packages/forms-web-app/__tests__/unit/controllers/examination/enter-comment.test.js

```diff
@@ -2,43 +2,179 @@ const {
 	getEnterComment,
 	postEnterComment
 } = require('../../../../src/controllers/examination/enter-comment');
-const { mockReq, mockRes } = require('../../mocks');
+const { mockReq, mockRes, mockResponse } = require('../../mocks');
+
+const pathShortner = (obj, path) => path.split('.').reduce((value, el) => value && value[el], obj);
```

Comment:
would this be better as: previousValue, currentValue

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Reply:
Or this would no be necessary as comments below

Reply:
I can modify it to match this convention but as reduce parameters are done in order a dev should know what they mean